CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 - Use transaction in `update_blocked_mailinglist_contacts`. #4058
 - Remove `Sql.get_conn()` interface in favor of `.call()` and `.transaction()`. #4055
 - Updated provider database.
+- Remove metadata from avatars and images before sending #4037
+- Reduce and recode images to JPEG if they exceed limits on width or height #4037
```

Comment:
Maybe, but lets give it another try ^)
And as for transparancy, in https://github.com/deltachat/deltachat-core-rust/issues/3956 afaiu we agreed with @r10s that sacrificing transparency is better then sending huge images. Anyway a user can send an image as a file. And also most of PNGs (incl. screenshots) won't be recoded because 500K looks quite enough for them

Reply:
images with viewtype sticker should't be recoded IMHO, they can be animated webp or APNG, but then they also have small width and height so probably don't get affected by this?

Reply:
ah, it got changed to >500k instead of width and height, I think an animated webp or APNG (even bigger sizes than webp) could be even 1MB I don't think it is good to mess with stickers, and you can't send them as files

Reply:
For stickers we use `Viewtype::Sticker` and for GIFs `::Gif`. So, they aren't affected, this code is for `Viewtype::Image` and avatars. But as for animated webp and APNG, u're right, i need to fix it. Thanks!